project4/styles.css

```diff
@@ -0,0 +1,490 @@
+    border-bottom-color: transparent;
+}
+
+nav {
+    -webkit-filter: drop-shadow(4px 4px 4px black);
+    filter: drop-shadow(4px 4px 4px black);
+    background-color: var(--color-offblack);
+    background-image: url("https://www.transparenttextures.com/patterns/low-contrast-linen.png");
+    height: 90px;
+    left: 0;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    z-index: 10;
+}
+
+span {
+    font-size: 24px;
+    font-weight: 900;
+}
+
+ul {
+    display: inline-block;
+    list-style: none;
+    text-align: center;
+    width: 100%;
+    margin: 0;
+    padding-left: 0;
+    position: relative;
+    top: 10px;
+}
+
+/*  ================== GLOBAL SETTINGS ================== */
+
+.button {
+    background: var(--color-offwhite);
+    border: 5px solid var(--color-offwhite);
+    color: var(--color-offblack);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    height: 70px;
+    margin: auto;
+    width: 200px;
+    padding-top: 15px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button:hover {
+    background: var(--color-lightgreen);
+    border: 5px solid var(--color-lightgreen);
+    color: var(--color-offblack);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    height: 70px;
+    margin: auto;
+    width: 200px;
+    padding-top: 15px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button:active {
+    background: inherit;
+    border: 5px solid var(--color-lightgreen);
+    color: var(--color-lightgreen);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    height: 70px;
+    margin: auto;
+    width: 200px;
+    padding-top: 15px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button-register {
+    color: var(--color-offwhite);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button-register:hover {
+    color: var(--color-offwhite);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: underline;
+    text-transform: uppercase;
+}
+
+.button-register:active {
+    color: var(--color-lightgreen);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: underline;
+    text-transform: uppercase;
+}
```

Comment:
Same note as above; you can lose most of these styles since they are reiterating the styles already on `.button-register`.